src/main/kotlin/me/clip/voteparty/cmds/CommandVoteParty.kt

```diff
@@ -29,16 +30,29 @@ internal class CommandVoteParty(override val plugin: VotePartyPlugin) : BaseComm
 {
 	
 	@Subcommand("addvote")
-	@Syntax("<amount>")
+	@Syntax("<amount> [player]")
 	@Description("Add Vote")
 	@CommandPermission(ADMIN_PERM)
-	fun addVote(issuer: CommandIssuer, @Default("1") amount: Int)
+	fun addVote(issuer: CommandIssuer, @Default("1") amount: Int, @Optional name: String?)
 	{
+		
 		if (amount <= 0)
 		{
 			return sendMessage(issuer, Messages.ERROR__INVALID_NUMBER)
 		}
 		
+		if (!name.isNullOrEmpty())
+		{
+			val user = party.usersHandler[name] ?: return sendMessage(issuer, Messages.ERROR__USER_NOT_FOUND)
+			
+			repeat(amount) {
+				user.voted()
+				user.claimable++
```

Comment:
shouldn't this only be increased if they're offline

Reply:
I think it should be either way, shouldn't it? Because what if they just want to give them votes in general? There's no specific reason they should be offline for it to work, right?